### PR TITLE
Added No members No Assingments in group page 

### PR DESCRIPTION
**Pull request by yash15601**
Fixes  https://github.com/CircuitVerse/CircuitVerse/issues/1024

#### Describe the changes you have made in this PR -
when their is No Member and No Assignments  in the group and assignments it should show some message like No Assignments, No Group

### Screenshots of the changes (If any) -
Before:
image

After:
image

### CODE REVIEW

**Comment by yash15601** [comment]
@tachyons  Review please

File: app/views/groups/show.html.erb [hunk 1]
```diff
@@ -20,6 +20,11 @@
       <h3>Members </h3>
       <button type="button" class="float-sm-left float-md-right float-lg-right float-xl-right add-member text-white" data-toggle="modal" data-target="#myModal">Add member</button>
       <br><br><br>
+      <% if @group.users.blank? %>
+      <div class="container h5 text-center">
+         No Members in the group,Add members
```
Comment (tachyons) [inline app/views/groups/show.html.erb]: @nitin10s Can you suggest better text here ?

**Comment by yash15601** [comment]
@nitin10s  okay

**Comment by yash15601** [comment]
@tachyons  Review please

**Comment by yash15601** [comment]
@tachyons review please

**Pull request merged**